### PR TITLE
PLANET-3726 fix control panel

### PR DESCRIPTION
**Pull request by angtheod**
Fix broken P4 Control Panel links. Ajax was failing. Update Check EN link to work once again after ENS API code was updated. Add check Elasticsearch connectivity link. Add Sync ES link. Remove old Check SearchWP indexer link.

### CODE REVIEW

File: classes/class-p4-control-panel.php [hunk 1]
```diff
@@ -247,28 +253,15 @@ public function check_search_indexer() {
...
-							$response['message'] = __( 'Indexer has stalled', 'planet4-master-theme-backend' );
-							$response['class']   = 'cp-error';
-						} else {
-							$response['message'] = __( 'Indexer is awake', 'planet4-master-theme-backend' );
-							$response['class']   = 'cp-success';
-						}
+				if ( wp_verify_nonce( $cp_nonce, 'cp-action' ) && 'check_elasticsearch' === $cp_action ) {
+					$response = [];
+
+					if ( ep_elasticsearch_can_connect() ) {
```
Comment (kirdia) [inline classes/class-p4-control-panel.php]: Looks good! This does not seem to work with latest elasticpress version, or is it happening only for me?

Comment (sagarsdeshmukh) [reply classes/class-p4-control-panel.php]: The "Check Elasticsearch" link is failing on my local as well - 
image

Comment (angtheod) [reply classes/class-p4-control-panel.php]: Fixed that. I had not updated to latest EP version, so it was working with 2.8.*

Comment (angtheod) [reply classes/class-p4-control-panel.php]: image

**Pull request merged**